cocotb/triggers.py

```diff
@@ -623,24 +608,20 @@ def __repr__(self):
         return "{}({!r})".format(type(self).__name__, self._coroutine)
 
 
-class Waitable(object):
+class Waitable(Awaitable):
     """
     Compatibility layer that emulates `collections.abc.Awaitable`.
```

Comment:
```suggestion
    Base class for trigger-like objects implemented using coroutines.
```
There's not really any compatibility going on here at all.

We may eventually want to remove this in place of using `Awaitable` directly, but there lie the dragons of distinguishing async coroutines from triggers / running tasks, which for now we should leave sleeping.